resources/metadata/mtaBuild.yaml

```diff
@@ -0,0 +1,29 @@
+metadata:
+  name: mtaBuild
+  description: Performs an mta build
+  longDescription: |
+    Performs an mta build
+spec:
+  inputs:
+    params:
+      - name: buildTarget
+        type: string
+        description: Lorem ipsum
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: false
+        default:
+  outputs:
+    resources:
+      - name: commonPipelineEnvironment
+        type: piperEnvironment
+        params:
+          - name: mtarFilePath
+            fields:
+              - name: mtarFilePath
+  containers:
+    - name: mta
+      image: ppiper/xs-cli
```

Comment:
use `devxci/mbtci`

Reply:
Was not updated --> early wip, thanks for the hint nevertheless.

Reply:
@CCFenner: is the new image `devxci/mbtci` also able to handle the old jar file based approach? Is there any need to distiguish which build tool is in fact used?